index.html

```diff
@@ -745,24 +745,28 @@ <h3>The <dfn>takeRecords()</dfn> method</h3>
     </p>
   </section>
   <section>
-    <h3>The <dfn>supportedSources</dfn> attribute</h3>
+    <h3>The <dfn>knownSources</dfn> attribute</h3>
     <p>
-      The {{PressureObserver/supportedSources}} attribute is informing on the [=supported source type=] by the [=platform collector=].
+      Each [=global object=] has an associated <dfn>frozen array of supported source types</dfn>, which is initialized
+      to the {{FrozenArray}} created from the sequence of strings values present in the enumeration
+      {{PressureSource}}, in alphabetical order.
```

Comment:
I find this confusing, although I can see it's similar to https://w3c.github.io/performance-timeline/#dom-performanceobserver-supportedentrytypes:
- Why is the data per-global object if the list of known sources is not going to vary per global?
- Please mention in the commit message that the list is now mandated to be returned in alphabetical order, this was not the case before.
- "A FrozednArray created from PressureSource" does not really say that it contains only the source types this implementation might have support for.

What if you just say something like "each user agent has a list of PressureSource values corresponding to the source types that are known to the implementation" and the getter returns this list in alphabetical order?

Reply:
I guess I can say user agent instead of global object given they are stable across user agent

Reply:
Actually I think this is wrong. It's a FrozenArray (ibject) and that needs to be attached to a JS realm, thus global object

Reply:
The internal representation of the list of supported sources does not need to be a FrozenArray, it just needs to be represented as such when the attribute getter is accessed.

In other words, I think you could simply mention in https://www.w3.org/TR/compute-pressure/#internal-slot-definitions that the user agent has a list of supported sources and, in the getter here, say that it should return the list in alphabetical order. Since the getter returns a FrozenArray, the conversion between a [list](https://infra.spec.whatwg.org/#lists) and a FrozenArray would be automatic.